### PR TITLE
Fixes #21 Add theme options to allow users to override urls in the menu bar

### LINKED ISSUE #21: Add theme options to override links on the menu bar
Currently, the links are hard coded in the theme. Instead of that it would be desirable if users have an option to override the urls

### PR DESCRIPTION
**Pull request by pri22296**

### CODE REVIEW

**Comment by pri22296** [comment]
@imujjwal96 @sch00lb0y Please review

File: sphinx_fossasia_theme/layout.html [hunk 1]
```diff
@@ -86,17 +86,33 @@
           <span class="icon-bar"></span>
           <span class="icon-bar"></span>                        
         </button>
+        {%- if theme_show_fossasia_logo|lower == "true" %}
         <a class="navbar-brand" href="#"><img class="img-responsive" src="http://fossasia.org/img/fossasia-dark.png" style="width: 50%" /></a>
+        {%- endif %}
```
Comment (imujjwal96) [inline sphinx_fossasia_theme/layout.html:91]: if false?

Comment (pri22296) [reply sphinx_fossasia_theme/layout.html:91]: Then nothing would be shown.

**Review by imujjwal96 — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks. Please fix travis in subsequent PRs. Cheers.

**Pull request merged**